src/ui/map.js

```diff
@@ -2618,16 +2622,36 @@ class Map extends Camera {
         return this.style.getFeatureState(feature);
     }
 
-    _containerDimensions() {
-        let width = 0;
-        let height = 0;
+    /*
+    * Returns height and width of map container.
+    * @private
+    * @returns {object}
+    */
+    _getContainerDimensions(): {width: number, height: number} {
+        return {width: this._containerWidth, height: this._containerHeight};
```

Comment:
Nit: we could use `map._containerWidth` and `map. _containerHeight` directly instead of adding one more method that wraps them to simplify the code — we always unwrap the values when using them anyway.